apps/nextjs-website/src/components/molecules/WebinarPlayerSection/WebinarPlayerSection.tsx

```diff
@@ -21,25 +21,26 @@ import VideoJsPlayer from '@/components/atoms/VideoJsPlayer/VideoJsPlayer';
 type WebinarPlayerSectionProps = {
   webinar: Webinar;
   webinarState: WebinarState;
-  isPlayerVisible?: boolean;
+  showQuestionBox?: boolean;
 };
 const WebinarPlayerSection = ({
   webinar,
   webinarState,
-  isPlayerVisible = false,
+  showQuestionBox = false,
```

Comment:
Rename to something like that:
```suggestion
  enableQuestionForm = false,
```